packages/react-performance/README.md

```diff
@@ -17,30 +17,9 @@ $ yarn add @shopify/react-performance
 
 The most basic way to use the tools in this package is to record information and display it locally to the user. In practice you usually only want to do this in development so that developers can easily see performance information.
 
-#### Add the context provider
-
-Before we can use any of the components or hooks in the package we must wrap our app tree with the `PerformanceContext.Provider` component.
```

Comment:
Oh weird we don't need this at all anymore?

Reply:
Ya we don't because performance object is created by default.
This *feels* weired but work pretty nicely when I :tophat: